devicetypes/smartthings/zwave-motion-sensor.src/zwave-motion-sensor.groovy

```diff
@@ -300,6 +354,87 @@ private command(physicalgraph.zwave.Command cmd) {
+			state.configured = false // this flag needs to be set to false when settings are changed (and the device was initially configured before)
+			if (!state.temperatureConfigured || temperatureAndHumidityReport != state.temperatureAndHumidityReport) {
+				state.temperatureConfigured = false
+				result << zwave.configurationV2.configurationSet(parameterNumber: 1, size: 2, scaledConfigurationValue: temperatureAndHumidityReport)
+				result << zwave.configurationV2.configurationGet(parameterNumber: 1)
+			}
+			if (!state.intervalConfigured || retriggerIntervalSettings != state.retriggerIntervalSettings) {
+				state.intervalConfigured = false
+				result << zwave.configurationV2.configurationSet(parameterNumber: 2, size: 2, scaledConfigurationValue: retriggerIntervalSettings)
+				result << zwave.configurationV2.configurationGet(parameterNumber: 2)
+			}
+		}
+	} else if (isEverspringSP817()) {
+		if (!state.retriggerIntervalSettings) state.retriggerIntervalSettings = everspringDefaults[2]
+		Integer retriggerIntervalSettings = (settings.retriggerIntervalSettings as Integer) ?: everspringDefaults[2]
+		if (!isConfigured() || (retriggerIntervalSettings != state.retriggerIntervalSettings)) {
+			result << zwave.configurationV2.configurationSet(parameterNumber: 4, size: 2, scaledConfigurationValue: retriggerIntervalSettings)
+			result << zwave.configurationV2.configurationGet(parameterNumber: 4)
+		}
+	}
+	return result
+}
+
+def getEverspringDefaults() {
+	[1: 600,
+	2: 180]
+}
+
+def lateConfigure() {
+	log.debug "lateConfigure"
+	sendHubCommand(commands(getConfigurationCommands(),200))
+}
+
+private isConfigured() {
+	return state.configured == true
+}
+
+private setConfigured() {
+	state.configured = true
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.configurationv2.ConfigurationReport cmd) {
+	if (isEverspringSP815andSP817()) {
+		if (cmd.parameterNumber == 1) {
+			state.temperatureAndHumidityReport = scaledConfigurationValue
+			state.temperatureConfigured = true
+		} else if ((isEverspringSP815() && cmd.parameterNumber == 2) || (isEverspringSP817() && cmd.parameterNumber == 4)) {
+			state.retriggerIntervalSettings = scaledConfigurationValue
+			state.intervalConfigured = true
+		}
+		if (state.intervalConfigured == true && (isEverspringSP817() || (isEverspringSP815() && state.temperatureConfigured== true))) {
+			setConfigured()
+		}
+		log.debug "Everspring Configuration Report: ${cmd}"
+		return [:]
+	}
+}
+
 private isEnerwave() {
 	zwaveInfo?.mfr?.equals("011A") && zwaveInfo?.prod?.equals("0601") && zwaveInfo?.model?.equals("0901")
 }
+
+private isEverspringSP815() {
+	zwaveInfo?.mfr?.equals("0060") && zwaveInfo?.model?.equals("0004")
+}
+
+private isEverspringSP817() {
+	zwaveInfo?.mfr?.equals("0060") && zwaveInfo?.model?.equals("0006")
+}
+
+private isEverspringSP815andSP817() {
```

Comment:
nit: I would change this to `or` instead of `and`

Reply:
sure, fixed